### ISSUE TITLE
Consider reworking the income a little bit

### ISSUE THREAD

**Issue by Folas1337** [Meta-Union/Meta-Union#29]
The general idea here is that it is a little unrewarding to do things on survival, you can do anything or nothing. While this allows you to freely do whatever you want, some people need a little motivation every now and then.
I just came to that conclusion while talking to Lena and Ice who said (partially jokingly) that they're missing Jobs as they were on the old survival server.

The main cause seems to be that it's not quite as rewarding if you do anything on survival. You do get skill exp but money would be cool as well, especially if it came in little bursts from reaching certain milestones.

Some ideas:
- Jobs
- Weekly quests
- "Regular quests" (like breaking 100 oak logs, breaking 100 birch logs, breaking 1000 stone etc.)

In other words: *A little more dopamine :>*

**Comment by Folas1337** [comment]
This has definitely turned into more than I thought... and it also evolved into this quicker than anticipated 😂 
I am now going to combine quests, daily quests and jobs in a pretty cool way (if you ask me).

There will be quests like "Kill 50 Zombies", then "Kill 100 Zombies", "Kill 200 Zombies" and so on up to "Kill 800 Zombies". The quests up until then all build on Minecraft's statistics, meaning if you killed the first 50, you do not have to kill another 100 to get the next but only 50 and then 100 for the "Kill 200 Zombies" quest.

The last one will be "Kill 1,000 Zombies" which will be a repeating quest and it starts at 0, so it is not building on the previous numbers/on the Minecraft statistics! This quest will be repeatable indefinitely and basically be the part where it turns into a kind of job as well as "weekly quest".

The general formula for money is the following for the 9 quests I will be having: 100 -> 150 -> 200 -> 250 -> 300 -> 350 -> 400 -> 450 -> 500 -> 2000

Skill XP formula is the following: 1k -> 1.5k -> 2k -> 2.5k -> 3k -> 3.5k -> 4k -> 4.5k -> 5k -> 10k

Although both formulas are subject to change depending on what feedback I'm getting.

Also they are 9 quests because that's exactly how many items you can add to 1 line in a chest and if I ever add more mob types like husks in the future, it would nicely add another line.

**Comment by Folas1337** [comment]
I've been working on this for a while and figured out how to add new quests constantly so I'll close this, although I'm not sure about each skill yet but I'll find things that work 😅
